### PR TITLE
CPFA-39 - Adding pages structure

### PR DESCRIPTION
**Pull request by MuscularSloth**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
+
 import './App.scss';
-import Home from './components/Home/Home';
-import Login from './components/Login/Login';
-import { BrowserRouter, Navigate, Route, Routes } from 'react-router-dom';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import HomePage from './pages/HomePage/HomePage';
+import NavigationBar from './components/NavigationBar/NavigationBar';
+import AuthorPage from './pages/AuthorPage/AuthorPage';
+import SinglePoemPage from './pages/SinglePoemPage/SinglePoemPage';
```
Comment (alexiStructShare) [inline src/App.js:7]: I would not add smth before the feature is started..

Comment (MuscularSloth) [reply src/App.js:7]: We discuss this at the meeting. We have decided that I must create whole structure.

**Review by KenAdams777 — APPROVED** [review]
Great job, comrade!
Looks reasonable and, in my humble opinion, this kind of infrastructure might be useful.

File: src/App.js (hunk 1 above)
Comment (bejalane) [reply src/App.js:7]: Ok. But it should be just an example, no need to create all the components.

**Review by bejalane — APPROVED** [review]

**Pull request merged**